### ISSUE TITLE
BUG: Unable to use 'the page should be audited for accessibility according to' - undefined step

### ISSUE THREAD

**Issue by nicholaspshaw** [dequelabs/axe-core-gems#428]
**Describe the bug**

**Environment Happening In**
- [X] Production

**To Reproduce**
1. Use, 'Then the page should be audited for accessibility according to: wcag21aa' within a test
2. Receive error, 'Undefined step: "the page should be audited for accessibility according to: wcag21aa" (Cucumber::Core::Test::Result::Undefined)'

**Expected behavior**
I'm able to use the step, 'the page should be axe clean according to: wcag21aa' without issue.  However, some places of our application require work so I'd like the output, without failing tests.  The following resource suggested this would be possible - https://docs.deque.com/devtools-for-web/4/en/rb-cucumber

**Actual behavior**
Test failed with undefined step, shared full error above.

**Screenshots**
N/A

**Related Tickets**
List out any user story or bug ticket(s) related to this bug. 

**Desktop (please complete the following information):**
- Product: [e.g. selenium] axe-core-cucumber
- Product Version: [e.g. v4.7.4] 4.10.2
- Latest Product Version it is NOT happening in: [e.g. v4.7.2] Unknown
- OS: [e.g. iOS] Sonoma 14.7
 - Browser [e.g. chrome, safari] chrome
 - Browser Version [e.g. 22]  130.0.6723.117

**Additional context**
I realise a discussion forum would be a better place but I cannot find this tooling discussed anywhere in depth

**Comment by nicholaspshaw** [comment]
I should have added, I don't see the step 'the page should be audited for accessibility' available within the code.  My confusion comes as the deque documentation seems to suggest you can do this.

Furthermore, the ability to audit the page i.e. checking against the ruleset and outputting violations without failing the test would be valuable to us.

**Comment by pedrolucasp** [comment]
>I should have added, I don't see the step 'the page should be audited for accessibility' available within the code. My confusion comes as the deque documentation seems to suggest you can do this.

Matter of fact, the docs seem to be AI-written or something for the Ruby gems, since they indicate stuff like:

`.skipping(:color-contrast)` which doesn't work since dashes are not allowed as symbols. Furthermore, the docs indicate a gem that doesn't exist or isn't available.
